lambdas/ec2/reaper.py

```diff
@@ -11,10 +10,26 @@
 
 ec2 = boto3.resource('ec2')
 
+def determine_live_mode():
+    """
+    Returns True if LIVE_MODE is set to true in the shell environment, False if
+    set to false or if not set at all. Raises error if LIVE_MODE is set any
```

Comment:
I wasn't sure if 'Raises error if LIVE-MODE is set any other string' had a typo and you meant 'is set as any other string' or 'is set to any other string'.